components/Image/ui-world-map.tsx

```diff
@@ -228,39 +238,90 @@ export default function UIWorldMap({ section }: { section: uiTypes.UISectionObj
     setIsDragging(false)
     setIsFirstPress(true)
     setAccel(1)
+    setTouchDist(0)
     console.log('Mouse LEAVE Event function')
   }
+  const moveLeft = () => {
+    console.log(`Image scale for moveleft is: ${imgScale}`)
+    /*     let xDiff = -50/imgScale
+    console.log(`xdiff is : ${xDiff}`)
+    if (imgLeft + xDiff <= leftLimit) {
+      setImgLeft(leftLimit)
+    } else if (imgLeft + xDiff >= 0) {
+      setImgLeft(0)
+    } else setImgLeft(imgLeft + xDiff) */
+    setImgLeft(-2084)
```

Comment:
Move to a constants file. This code appears to be duplicated when compared to your new-map-test. I'm seeing the same numbers/code. Did you mean for this to happen?

Reply:
This component will be delete and code in ui-sectioned changed